specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/secrets.json

```diff
@@ -0,0 +1,491 @@
+          "description": "The attributes of the secret."
+        }
+      },
+      "description": "Properties of the secret",
+      "type": "object"
+    },
+    "SecretAttributes": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/Attributes"
+        }
+      ],
+      "description": "The secret management attributes.",
+      "type": "object"
+    },
+    "SecretCreateOrUpdateParameters": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags that will be assigned to the secret. "
+        },
+        "properties": {
+          "$ref": "#/definitions/SecretProperties",
+          "description": "Properties of the secret"
+        }
+      },
+      "description": "Parameters for creating or updating a secret",
+      "required": [
+        "properties"
+      ],
+      "x-ms-azure-resource": true,
+      "type": "object"
+    },
+    "SecretPatchParameters": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags that will be assigned to the secret. "
+        },
+        "properties": {
+          "$ref": "#/definitions/SecretPatchProperties",
+          "description": "Properties of the secret"
+        }
+      },
+      "description": "Parameters for patching a secret",
+      "x-ms-azure-resource": true,
+      "type": "object"
+    },
+    "Secret": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SecretProperties",
+          "description": "Properties of the secret"
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "allOf": [
+        {
+          "$ref": "./keyvault.json#/definitions/Resource"
+        }
+      ],
+      "description": "Resource information with extended details.",
+      "type": "object"
+    },
+    "SecretListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Secret"
+          },
+          "description": "The list of secrets."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of secrets."
+        }
+      },
+      "description": "List of secrets",
+      "type": "object"
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client Api Version."
+    }
```

Comment:
You are highly recommended to remove your own definitions of these and use the definitions from common-specification/types/v3/types.json instead

Reply:
This is old existing file, not introduced in this PR, so skipping.